### PR TITLE
port ugent infra page to user docs

### PR DESCRIPTION
**Pull request by wdpypere**
not super happy with the tables, apparently mkdocs does not support code line splitting for tables. but here it can be managed by multiple people, not like the current page.

### CODE REVIEW

File: mkdocs/docs/HPC/infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+The Stevin computing infrastructure consists of several Tier2 clusters
+which are hosted in the S10 datacenter of Ghent University.
+
+This infrastructure is co-financed by FWO and Department of Economy,
+Science and Innovation (EWI).
+
+## Tier-2 login nodes
+
+Log in to the HPC-UGent Tier-2 infrastructure using SSH via `login.hpc.ugent.be`.
```
Comment (stdweird) [inline mkdocs/docs/HPC/infrastructure.md:13]: or visit URL ...

Comment (stdweird) [reply mkdocs/docs/HPC/infrastructure.md:13]: maybe beter visit uRL .. or SSH ..

Comment (wdpypere) [reply mkdocs/docs/HPC/infrastructure.md]: fixed

**Comment by wdpypere** [comment]
ok, now the tests passed. only error is github.com getting blocked on the mathlab pages. but when I check manually, it works just fine.

**Comment by boegel** [comment]
> ok, now the tests passed. only error is github.com getting blocked on the mathlab pages. but when I check manually, it works just fine.

@wdpypere The whitelist for the link checker was updated in #765, so you should sync with `main` branch to fix/ignore this

**Comment by wdpypere** [comment]
@boegel , rebased.

**Pull request merged**